lib/src/wallet.rs

```diff
@@ -632,7 +632,7 @@ impl LightWallet {
             .unwrap_or_default()
     }
 
-    async fn get_target_height(&self) -> Option<u32> {
+    async fn get_wallet_latest_height(&self) -> Option<u32> {
```

Comment:
This function returns 1 + latest height, so calling it `wallet_latest_height` is incorrect.

Reply:
Wait...   why does this function return 1 + last_synced_height?

Reply:
Yes.